designs/node-repair.md

```diff
@@ -18,99 +18,145 @@ The alpha implementation will not consider these features:
 
   - Disruption Budgets
   - Customer-Defined Conditions
-  - Customer-Defined Remediation Time
 
 The team does not have enough data to determine the right level of configuration that users will utilize. The opinionated mechanism would be responsible for defining unhealthy notes. The advantage of creating the mechanism would be to reduce the configuration burden for customers. **The feature will be gated under an alpha NodeRepair=true feature flag. This will  allow for additional feedback from customers. Additional feedback can support features that were originally considered out of scope for the Alpha stage.**
 
-## Recommendation: CloudProvider-Defined RepairPolicies
+## Recommendation: CloudProvider-Defined RepairPolicies with NodePool-Configurable TolerationDuration
```

Comment:
Can you write a separate section in this doc or move the changes you want to make to a separate design? It will make it hard to track in the future all the design choice and consideration we have made